src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentContext.cs

```diff
@@ -4,14 +4,15 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor;
 using Microsoft.AspNetCore.Razor.Language;
-using Microsoft.AspNetCore.Razor.Language.Syntax;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
-using Microsoft.CodeAnalysis.Razor.Workspaces;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer;
+namespace Microsoft.CodeAnalysis.Razor.Workspaces.ProjectSystem;
+
+using Microsoft.AspNetCore.Razor.Language.Syntax;
```

Comment:
Why is this needed?

Reply:
Now that the code is in a `Microsoft.CodeAnalysis.*` namespace, things like `TextSpan`, `SyntaxNode` etc. now match to the Roslyn type first, over the Razor type. In this files' case its the `SyntaxNode` return from `GetSyntaxNodeAsync`.

Reply:
Ah, that pesky `Microsoft.CodeAnalysis.*` namespace. Got it.